crates/proto/src/tcp/mod.rs

```diff
@@ -15,9 +15,10 @@
  */
 
 //! TCP protocol related components for DNS
-
+#[allow(clippy::all)]
```

Comment:
can we narrow this to just the errors that clippy raised?

Reply:
Sure, I just updated a new patch.

Reply:
I don't see the change, did you push it?